Core/source/test/test-utils.ts

```diff
@@ -141,12 +141,19 @@ const TEST_KEYS: { [name: string]: TestKey } = {
+    decrypted: '',
+    passphrase: 'London blueBARREY capi',
+    longid: '4B5A2FFCE62D9501'
```

Comment:
This is not the primary longid of this key (can check with gnupg / pgpdump or our own lib). The primary fingerprint is `E8F0517BA6D7DAB6081C96E4ADAC279C95093207` which makes the longid `ADAC279C95093207`